debian/changelog

```diff
@@ -1,3 +1,9 @@
+privacyidea-ucs (2.19.1-1) unstable; urgency=medium
```

Comment:
I updated the PR. This should work. https://dyn.manpages.debian.org/jessie/dpkg/dpkg-maintscript-helper.1.de.html